reset-account-nonce-script/.env.example

```diff
@@ -0,0 +1,6 @@
+DELETABLE_ACCOUNT_PK=    # hex encoded
+
+OPERATOR_ID=             # format 0.0.9999
```

Comment:
Add a comment for EVM devs to show how from their Operator ID from their evm address
Actually better yet i don't think we should use this. 
Let the evm user provide their address and let the code pick up the Operator ID

Reply:
I got it and fixed the script to be used from an EVM dev perspective, not a Hedera one. Actually, the operator ID or address is not needed because it can be derived by the operator's private key and the query the mirror node its account ID.